torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20884,6 +20906,11 @@ def __init__(
         torch_opinfo_name="conj",
         supports_nvfuser=False,
     ),
+    PythonRefInfo(
```

Comment:
Perfect. 

cc @IvanYashchuk -- why doesn't this work with nvFuser?

Reply:
Probably `narrow`

Reply:
`torch.ops.prims.slice_in_dim` doesn't have nvFuser support yet.